src/pages/costModels/createCostModelWizard/index.tsx

```diff
@@ -223,7 +224,15 @@ class CostModelWizardBase extends React.Component<Props, State> {
           description: this.state.description,
           onDescChange: value => this.setState({ description: value }),
           markup: this.state.markup,
-          onMarkupChange: value => this.setState({ markup: value }),
+          onMarkupChange: value => {
+            const markupDecimal = Number(value)
+            const dx = value.split('').findIndex(c => c === '.')
+            if(!isNaN(markupDecimal) && dx > -1 && value.length - dx - 1 > 2) {
+              this.setState({markup: formatValue(markupDecimal, 'markup', {fractionDigits: 2}) as string})
+              return
+            }
+            this.setState({ markup: value });
+          },
```

Comment:
Not clear as to what changed, but this code still appears to split a string? Not all locales will use a decimal